packages/esm-ward-app/src/ward-view/ward-view.resource.ts

```diff
@@ -86,5 +97,6 @@ export function createAndGetWardPatientGrouping(
     wardPatientPendingCount,
     bedLayouts,
     wardUnassignedPatientsList,
+    totalPatientsCount: wardAdmittedPatientsWithBed.size + wardUnadmittedPatientsWithBed.size,
```

Comment:
There's also another group here: admitted patients without beds... @chibongho  may potentially know  how to get this metric, I'd have to dig deeper into the code to refresh my memory.

Reply:
Oh k i will connect @chibongho . I think it might be the wardUnassignedPatientsList ,but i will confirm with @chibongho once.